### ISSUE TITLE
Variants not being Displayed in URL

### ISSUE THREAD

**Issue by JakeHenshall** [Hujjat/shopify-starter-theme#1]
Hi,

Great theme and Youtube course I'm trying to learn by using your theme but I'm facing an issue.

Please find attached a video of the issue.

https://imgur.com/Ho81kuS

Hopefully, you can help with a fix for this.

Jake.

**Comment by Hujjat** [comment]
Hey Jake, 

It's not required to add variant id on the url. May I know why you need it ?

**Comment by JakeHenshall** [comment]
@Hujjat Again thanks for answering. This issue is if I have 1 product with 2 variants. Those variants have different prices, Variant A = 9 and Variant B = 15, I think the price needs to change to whichever the selected variant is. If I select and add to cart Variant B it does add the correct one, it just doesn't update the price on the product page. 

![price1](https://user-images.githubusercontent.com/4720782/73399106-96e53c80-42de-11ea-89a3-3237dd5584c8.png)

![price2](https://user-images.githubusercontent.com/4720782/73399111-99e02d00-42de-11ea-8d8a-a42c8eed9b5f.png)

![price3](https://user-images.githubusercontent.com/4720782/73399127-9e0c4a80-42de-11ea-8aef-f0075c0569bc.png)

**Comment by nikolay-gavrilov** [comment]
Yes! I confirm. Have the same problem :( Are there any working options?
